### PR TITLE
Eventing v0.5 updates

### PR DESCRIPTION
**Pull request by samodell**
Part of #1020

### CODE REVIEW

File: docs/eventing/README.md [hunk 1]
```diff
@@ -39,9 +39,24 @@ generic interfaces that can be implemented by multiple Kubernetes resources:
...
    external event source are processed.
 
+### Event brokers and triggers
+
+As of v0.5, Knative Eventing defines Broker and Trigger objects to make it
+easier to filter events.
+
+A Broker provides a bucket of events which can be selected by attribute. It
+receives events and forwards them to subscribers defined by one or more matching
+Triggers. Each namespace has a default Broker, and for most use cases, no
```
Comment (Harwayne) [inline docs/eventing/README.md]: The namespace only has a default Broker if it is annotated. If it is not annotated, no default Broker is created (although users are free to create it themselves).

Comment (samodell) [reply docs/eventing/README.md]: Ah, okay. Deleted that sentence altogether.

**Comment by samodell** [comment]
Ready for a second look, @Harwayne

File: docs/eventing/README.md [hunk 2]
```diff
@@ -73,25 +87,6 @@ subscription, and if you have any functions that make use of any external (to
 cluster) services, you must enable it also for them to work.
 [Follow the instructions to configure outbound network access](../serving/outbound-network-access.md).
 
-Install the core Knative Eventing (which provides an in-memory
```
Comment (Harwayne) [inline docs/eventing/README.md]: Why are these instructions removed? Are they now in-lined in "Fllow the instructions to install on the platform of your choice"?

Comment (samodell) [reply docs/eventing/README.md]: Yep, the install guides cover install steps for all the components (Eventing included). There's no reason to repeat the content on this page.

**Pull request merged**